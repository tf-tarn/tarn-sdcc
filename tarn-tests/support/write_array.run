./sim/sim -q -n -s 1000 -h $program_file > $output
