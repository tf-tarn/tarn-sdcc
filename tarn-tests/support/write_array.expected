OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=00
OUTPUT=35
OUTPUT=34
OUTPUT=33
OUTPUT=32
OUTPUT=31
OUTPUT=05
